datasets/basiskaart/basiskaart.json

```diff
@@ -0,0 +1,490 @@
+{
+  "type": "dataset",
+  "id": "basiskaart",
+  "title": "Basiskaart",
+  "description": "Gegevens uit BGT (basisregistratie grootschalige topografie) en KBK (kleinschalige basis kaart)",
+  "license": "public",
+  "status": "beschikbaar",
+  "version": "0.0.1",
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam",
+  "authorizationGrantor": "OIS", 
+  "keywords": ["GBT", "KBK", "KBK10", "KBK50"],
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "gebouwVlak",
+      "title": "Gebouw vlak",
```

Comment:
Klein ding: 'gebouwvlak' is toch 1 woord?

Reply:
Ja klopt, eigenlijk is vlak, punt, lijn e.d. een typering van het onderwerp. Dus vandaar ik deze als los woord had toegevoegd. Misschien beter om dit toch als 1 woord te zien. Ook mooier in de URL van de API. Ik pas het aan!